units/src/locktime/relative.rs

```diff
@@ -178,4 +185,33 @@ mod tests {
         let result = Time::from_seconds_ceil(maximum_encodable_seconds + 1);
         assert!(result.is_err());
     }
+
+    #[test]
+    fn from_seconds_floor_success() {
+        let actual = Time::from_seconds_floor(100).unwrap();
+        let expected = Time(0_u16);
+        assert_eq!(actual, expected);
+    }
+
+    #[test]
+    fn from_seconds_floor_with_exact_interval() {
+        let actual = Time::from_seconds_floor(512).unwrap();
+        let expected = Time(1_u16);
+        assert_eq!(actual, expected);
+    }
+
+    #[test]
+    fn from_seconds_floor_with_maximum_encodable_seconds_success() {
+        let maximum_encodable_seconds = u16::MAX as u32 * 512;
```

Comment:
`maximum_encodable_seconds` could just be a constant  `MAXIMUM_ENCODABLE_SECONDS` instead of re-defining it in every test.

Reply:
good call, i'll pull it out into the test scope